src/ios/Default-Info.plist

```diff
@@ -3,7 +3,7 @@
 <plist version="1.0">
 <dict>
         <key>CFBundleIdentifier</key>
-        <string>com.gluonhq.otn.OTNApplication</string>
+        <string>DevoxxApplication</string>
```

Comment:
I believe this must be the fully qualified class name, including the package. You should also change the name of the application in de CFBundleExecutable and CFBundleName keys further down.